### DISCUSSION TITLE
I cannot delete and hide comments

### DISCUSSION THREAD

**Discussion by manfwh** [community/community#60022] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

[video.webm](https://github.com/community/community/assets/23977810/a7b5bdaf-aef0-4c2d-b9f2-d8d1027b638d)

network
![screenshot-20230704-144827](https://github.com/community/community/assets/23977810/ec8b971b-d5e0-43bd-a3ef-96445ed19daf)

**Comment by GhostOf0days** [comment]
Btw, if you contact [Github Support](https://support.github.com/), they can remove it for you. You can create a ticket by clicking 'My Tickets' in the top right corner. They can even do stuff like clear the repos you viewed from appearing in the searchbar and wiping the deleted repos in your account, so they no longer exist (this makes it unrecoverable).

**Reply by AnkanSaha** [reply]
remove it
